CsStat.Web/assets/components/base-layout/scss/base-layout.scss

```diff
@@ -25,4 +26,24 @@
             margin: 0;
         }
     }
+
+    &__footer {
+        background: lighten($bg-color-main, 10%);
+        padding: 20px 0 0;
+        color: $color-white;
+        &_copyright {
```

Comment:
all elements should be defined on new line, use dash here: 
&__footer-copyright {}